packages/cadl-ranch-specs/http/payload/content-negotiation/mockapi.ts

```diff
@@ -56,7 +56,7 @@ Scenarios.Payload_ContentNegotiation_DifferentBody = withKeys(["image/png", "app
           pass: "application/json",
           status: 200,
           body: json({
-            value: pngFile.toString("base64"),
+            content: pngFile.toString("base64"),
```

Comment:
tsp is `content`
https://github.com/Azure/cadl-ranch/blob/0c55717adbfcec16915121b39008d5ec4551a1da/packages/cadl-ranch-specs/http/payload/content-negotiation/main.tsp#L55-L58